tests/pset.rs

```diff
@@ -49,6 +54,51 @@ mod tests {
         psbt_rtt(&elementsd, &psbt_base64);
     }
 
+    #[cfg_attr(feature = "integration", test)]
+    fn tx_issuance() {
+        let (elementsd, _bitcoind) = setup(false);
+
+        let address_asset = elementsd.get_new_address();
+        let address_reissuance = elementsd.get_new_address();
+        let address_lbtc = elementsd.get_new_address();
+        let prevout = elementsd.get_first_prevout();
+
+        let contract_hash = ContractHash::from_inner([0u8; 32]);
+        let entropy = AssetId::generate_asset_entropy(prevout, contract_hash);
+        let asset_id = AssetId::from_entropy(entropy.clone());
+        assert_eq!(
+            asset_id.to_string(),
+            "78b4920005fa0156ae3779129338bc2707e4d07cf8a0c2593583e3c1da3bb58c"
```

Comment:
Not sure what's best here:
1) remove the asserts
2) add a comment about the possible change in upcoming elements release
3) I wanted to add something like `if cfg!(all(feature = "elementsd/0_21_0", feature = "elementsd/0_18_1_12"))`, but it looks rust doesn't support this unless we add features like `elementsd_0_21_0 = ["elementsd/0_21_0"]` which is probably not worthy for this, but it may be useful to have for other integration tests

Reply:
We are (indirectly) asserting that the utxo (which is the first one obtained from `listunspent`) is a specific utxo with a fixed hard-coded outpoint. That might change in elements release because the order of coins in listunspent is updated (I am not sure if the ordering is guaranteed to be the same across multiple calls of elementsd of the same version). 

I am okay adding a comment here if you think there is some value in asserting the fixed asset_id (partly because I am curious to see if this ever breaks :) ). But if this ever breaks in the future, I think we should just remove it.

Reply:
I just removed the asserts in 682eb8a5bc972e28ccea8997ec582aa2d28769a1, they weren't really asserting something in the scope of this PR